integrating

```diff
@@ -11,29 +11,27 @@ tips and advice to prove a smooth integration process.
 
 When you want to submit a change into illumos, we call that a request to
 integrate, or RTI. illumos has no single person that serves as a gatekeeper. An
-RTI is sent to the advocates. An advocate who is familiar with the subject
-matter will review the materials and either accept or reject the RTI. If it is
-accepted, the advocate will push those change into illumos. If it is rejected,
-the advocate will explain why, for example inadequate testing, and you can
-return when you have corrected those issues. The advocates is made up of members
-of the community and while they have commit access to the gate, they are also
-required to go through the same process.
+RTI is sent to the the core team. n core team member who is familiar with the
+subject matter will review the materials and either accept it or not. If the
+change isn't accepted for whatever reason, then the core team member will
+explain why and work with to get things integrated. Most of the time, the core
+team members are just looking for some small amount of additional explanation
+or confirmation about a change. The core team members go through this same
+process.
 
 ### The Requirements
 
-When you come to the advocates, you are required to have a few different pieces
-of information. We'll start off by listing them now, and go into more detail a
-bit later about each of them.
+When you want to integrate a change, it should include the following information:
 
   * List of Reviewers
 
-  * Explanation of Testing
+  * Explanation of Testing in the ticket
```

Comment:
why is testing capitalized?

Reply:
Probably because when I did this almost a decade ago I was trying to think of it like a section title ala reviewers above. Cleaned up.